com.sap.cloud.lm.sl.mta/src/main/java/com/sap/cloud/lm/sl/mta/model/NamedElementWithParametersContainer.java

```diff
@@ -0,0 +1,5 @@
+package com.sap.cloud.lm.sl.mta.model;
+
+public interface NamedElementWithParametersContainer extends NamedElement, ParametersContainer {
```

Comment:
Probably other name, e.g. NamerParametersContainer 
How this sounds?

Reply:
Renamed to -> NameParametersContainer